lib/src/json_schema_form.dart

```diff
@@ -1,30 +1,61 @@
+import 'dart:convert';
+
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_json_schema/src/checkbox_form_field.dart';
-import 'package:flutter_json_schema/src/json_schema_bloc.dart';
 import 'package:flutter_json_schema/src/models.dart';
 
 class JsonSchemaForm extends StatefulWidget {
-  final Schema schema;
-  final JsonSchemaBloc jsonSchemaBloc;
+  final String jsonSchema;
+  final String uiSchema;
+  final Function(String) onFormDataChanged;
 
-  JsonSchemaForm({@required this.schema, this.jsonSchemaBloc});
+  JsonSchemaForm(
+      {@required this.jsonSchema,
+      @required this.uiSchema,
+      this.onFormDataChanged});
```

Comment:
onFormDataChanged is not required but the other 2 params are. Is there a reason for this?

Reply:
I was thinking that the user can get the formData from the property whenever they want instead of depending on the callback. But I see that the "formData" is not there in JsonSchemaForm but in its state. I am down to change it to be required (not had strong feeling towards not to have it required).

Reply:
Lets make it required for now, I like the idea of being able to get the current data whenever as a property. I'll create an issue for that.